src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private Random random = new Random();
+    private double randomNumber;
+
+    public Figure getRandomFigure() {
+        getRandomFigureNumber();
+        ColorSupplier colorSupplier = new ColorSupplier();
+        if (randomNumber == 0) {
```

Comment:
use switch/case

Reply:
Ivan's comment not fixed, please use switch/case